frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "version": "2.2.1",
   "scripts": {
-    "prepare": "pnpm i18n:get-translations --en-only",
+    "prepare": "pnpm i18n:get-translations --en-only && cp -r test/locales/* src/locales",
```

Comment:
Is `prepare` only run for tests? If not, this will cause problems because we are copying the test locales (specifically generated for Playwright tests, Arabic, Russian, Spanish and English only).

Reply:
The `prepare` script is run everytime the package is set up (immediatey after `pnpm install`), but then running `pnpm i18n`, which we do before building the Docker images, will overwrite the test locales with the actual locales downloaded from GlotPress.